File: Tier2/src/main/java/group6/semester/project/model/Post.java
package group6.semester.project.model;

import com.google.type.DateTime;
import lombok.Data;
import org.springframework.stereotype.Component;

import java.time.LocalDate;
import java.util.List;
@Component
@Data

public class Post {
    private int id;
    private String title;
    private String description;
    public Date dateCreated;
    private int price;
    private String address;
    private String condition;
    private String phoneNumber;
    private String email;
    private List<Image> images;
    private User writer;



}

File: Tier1/Tier1/Entities/Models/Post.cs

namespace Entities.Models; 

public class Post
{
    public int Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public Date DateCreated { get; set; }
    public int? Price { get; set; }
    public string? Address { get; set; }
    public string Condition { get; set; }
    public string? PhoneNumber { get; set; }
    public string? Email { get; set; }
    public ICollection<Image> Images { get; set; }
    public User Writer { get; set; }
    

}